dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/hibernate/dialect/DhisH2Dialect.java

```diff
@@ -41,6 +41,7 @@
     public DhisH2Dialect()
     {
         registerColumnType( Types.JAVA_OBJECT, "text" );
+        registerColumnType( Types.JAVA_OBJECT, "jsonb" );
```

Comment:
What is this? did you mean `Types.JAVA_OBJECT-1` ?

Reply:
Yep, you're right. Changed it and apply codestyling https://github.com/dhis2/dhis2-core/pull/1453/commits/87a14361f13f627504c92fe75bcce14a3d519a2a